assets/scss/templates/_blog.scss

```diff
@@ -74,6 +74,23 @@
       line-height: 26px;
     }
 
+    ul {
+      color: #7B7B7B;
+      font-size: 15px;
+      padding: 10px 20px;
+      font-family: $primary-font;
+      list-style: circle;
+    }
```

Comment:
please remove this. We can not use ul color directly

Reply:
hi @mehedi-sharif 
just remove color and all the PR ?